subprojects/packagefiles/json-c/config/meson.build

```diff
@@ -72,6 +70,10 @@ if conf.has('HAVE_STRINGS_H')
   check_function += 'strncasecmp'
 endif
 
+if cc.has_function('arc4random')
```

Comment:
The loop below already calls `has_function` so the extra conditional isn't needed.